.buildconfig.yml

```diff
@@ -301,10 +301,6 @@ projects:
 #  service-pocket:
 #    path: components/service/pocket
 #    description: 'A library to communicate with the Pocket API'
-#    publish: true
-#  service-telemetry:
-#    path: components/service/telemetry
-#    description: 'A generic library for generating and sending telemetry pings from Android applications to the Mozilla telemetry service.'
```

Comment:
This is already back commented in, you may need to rebase before you can merge. :)